tests/constants.py

```diff
@@ -1,19 +1,18 @@
 # -*- coding: utf-8 -*-
 
 import logging
-import unittest
-from json import dumps
-
 import os
 import tempfile
-from requests.compat import urlparse
+import unittest
+from json import dumps
 
 from pybel import BELGraph
 from pybel.constants import *
 from pybel.manager import Manager
 from pybel.parser.parse_bel import BelParser
 from pybel.parser.parse_exceptions import *
 from pybel.parser.utils import any_subdict_matches
+from requests.compat import urlparse
```

Comment:
Again, optimize imports

Reply:
What do you mean by optimize?

Reply:
It's under reformat code. It just reorganizes the imports to look nice and gets rid of extra stuff that isn't used.

<img width="860" alt="screen shot 2017-10-26 at 5 59 21 pm" src="https://user-images.githubusercontent.com/5069736/32063444-784359ca-ba77-11e7-897a-c0e21841055b.png">